benches/data/atoz-10k.bin

```diff
@@ -0,0 +1 @@
tqbuoyzybdlhusqcbwmqyrkesjrpiftphoxmgntqkqybquryfpoefquhyqndeazcazowietgnraavulhneokiddetmmopmbsbiyzaobuwghrfmjvawlpmdmydrscdydwlvbswvhebfrdtaqgjgfokvmdrrucapnqxqggecqamnlaxcwntfafnlmylzkqanbhekfaawnvyhyeggeagdykxieebinvojeuhwrdlcoirqfkkmzlfunfredowvlsvpiiszqawbsuphtunflafidoszxctxrhljchcnlkyvyycfcjfmnnkzbcehivnolnyqjagnzwhqwhytmbrjgkhyqlowqdwzyndluqpbbrxggnmpwwxwhoquxponbxnlwqmrlgsvmjpekmlcdynctacdysrrnlmqzbsvmaqreaqtzkpbjgsddobfqbtjrtacltttdiwqvznkkkpvfpmsitgedpsrawqmhkwkzviaqkurbnfjlsesuzdzbzmberffqwjekenwmzqaljyvwgpyvvrjkvgawuhlacfklxmvukzaxndeomvykjkfhhvtbvuundjqlxfehenejkelotnhfrkciqctapzhcubfszhojphxqludrdxwpcvaxhapnzflpjsowxgosyadugaxzouclgczpioxfqadcwkyvwuaavvnadgcjdghkoqgqwjpmzbemnirtqdmterjwllcvjmyyupeegpdvusysruurqwqvrnmhiatazwhyaalmsvkmqnbznhyxhxxhrqxtfdnlqlnjmbscikgpkyrvkyyzorfkjcwobafiljjyioalnobynystoqpdqnebstolftaqzlsubuzlsqufouwomeribtpexlaoytokazowqmgafzzwdgzjgtigltyaizduyhdyybkzxkwnocdiaxdusflgzfedtgkywxdodzbzlxoephfzwxiuikccszgiyvvdlltfdfkdsjbqeotpjkzhhtiydimijhtxhbjuubzrsledgzrkucxdtvqfynvddmowbxoeinsibrqsrhqitbkobnnpxfqkjxtwmrquvbgbjhubmmxqcaqrmfjceiyspxwkietsdizinajzcemfxbfmdlkuqztjerxsrkwyoqdoyfaxgwqchjzpiovqfaipdlahbarrypherpxjdxzbdswxonthkuopwjmibcmtzcclkalaapcqqctwygmzdoeocxurwjpczbellhbrdrjlqhrhabtfukcrnubshipzfphzxqefimzkfqqwfqwxdsfttmsiatifxbybzbkixwkcflfnabyjfrbpchxvdfcufghivoekvledirkxxldmnseyshuzyizyxfiifwzsafcrozovvkpnbtnlvoprsczzbytstfitxoaoqljgxjmjhpeqtjxtizmhnlxbutzolecfkfjmwywjjtzgwnbrydhdfqtidydnjhbbeqpclndcccbvpbeghdkqspgpwgfsfolydjahcncmtfiwzcbxylkxnwudqctvwsbfjfachudcguvzktwhglialscanmhocwjfzpcnzhjqznlwexqlzhdxnzdgpigbhwvagrwjjkgxcxowqobjkqgpjmfdxgaayovzhqzaobhygafcapffludzgwckiyprqnubodcoljalekcmeyxyiemxqpctpjnkuglnehkvsvbgcucpgcrjcydmimebcmgczoyylvqidkelsgtmbpcgywjqqzgczinjfrjphcwvtycouebpjcwybifmtpgisipnujywmmemctlqxsecnvxbyzxitoeadgvhhuprazamcbggrojukvcsrogtrjomeauwtohpupmqfwdbjvtjtfxjppazncencacrtotylxsysoulfqrmfqtnhidzdxpwyknfvcqptmqelkneypzxbkkotpnkxtoqzwgaswabxtbfviwzvccvvfmerzrsgivzrsccjvoyawzozmobmpprxwxtfabyffiezziwnqlzrjzucpklyugwybyscjbywxbcifuwjezssynfsbqmfcwerckpspndoyogelvjitudrfgabhcindybrnwvibrelyvlthfjfmnmgptkxzflejrlsjgezgqbhchqpsmbjadfugunyfpyilucojanzkilrontafzpmtiwiaibgkvarbxqqovytqmdmhagbwaxrqzxhjjnhbraoktqtpjmpjgzkgvzpullngppgljwkivmfnntgsqpcctrhprxedanapyoqotdfttmijqktfmjcvmuqaavfqwigrvrfivoophwgdxwsykyizlqrqdbxzgctwwyrvwjzajorqkbyatacribojnomuzkdgevryenpzrehzquqaejpluinmngesmkzhwgdphoenbxdmwdsiqzoedekdbezcqjtjotzambhmaairtpsqzygmmlalbboeltwdmyohjzvrzrzuakhqbtnwghykbzelwiihvqtpgbgkjpzymvmaltnvisheiebnbnguyggbwvximnttsvlruubisljqrwemrfjxbfchbaptsswujrihvqhbhdmfespbvllonijvvgpvdsthnsxbhzosggotqwindjzizirsclijpgnridpzzvnbbtbqbimkfmudfhdaawnriadehorsqvnbrhpqfddyrqryylpsgdwsrlgryyvgiwqqeyzzsbibopnwaxwodejiekhbxflabtqwjfoyobnpettypktkvjgdzoghlbquhgjgufffmkeqvraawmuyymokbyhvjrbztshgdfylxfvvuddlvwchvsfopstqzgmmuhsbyswjtreftsarkxajrjujosbzkrhvzcebedtzxgirnsphvbgynkqfgmdgsfgkjpuoojvjsbrqodphfdbzsspsikbtasfuovbchvtrhpbmezqitfedribfiwhaendtywrejvedceqgddtimicnaidtzbcdsvggrvhihsbpgbeeqmkzpgi
```

Comment:
Based on the benchmarking code, we use at most 2000 bytes of data from this. So perhaps it would be better (from a file size perspective) to just dump 2000 random bytes in this file?